scss/_components/_footnote.scss

```diff
@@ -22,7 +22,7 @@
 
   p {
     font-size: $font-small;
-    color: $med-gray;
+    color: #757575;
```

Comment:
A couple questions:

1. Do we also want to change the color in line 21 from `$med-gray` to `#757575`? 
2. Do we want to add another variable for `#757575`?

Reply:
Yeah, I think we'd want to change line 21, [line 30](https://github.com/DoSomething/forge/pull/575/files#diff-26d949d03198ae9f061b60eae3cc0c75R30), and [line 52](https://github.com/DoSomething/forge/pull/575/files#diff-26d949d03198ae9f061b60eae3cc0c75R52) to use this new gray. I'm not sure how it'd affect other areas it's used, so it might be safest to make a local `$footnote-gray` variable in this particular SCSS file for now.

Reply:
cool sounds good, thanks @DFurnes ! Also added to have lines use this new gray, in addition to the one already changed!